karma.conf.js

```diff
@@ -29,11 +59,7 @@ module.exports = function karma(config) {
     reporters: ['dots'],
 
     files: [
-      'test/*.{jsx,js}',
-    ],
-
-    exclude: [
-      'test/_*.{jsx,js}',
+      'test/*-spec.{jsx,js}',
```

Comment:
there was a reason for that change: the new file `test/react-compat.js` was, under the old code, treated as a test file, which is incorrect and fails the karma tests (one hopes)

so, either the new file is renamed to `_react-compat` (which is strange but fine with me) or we accept that all our test files indeed have `-spec` in the filename (which is the case so far) and make that the rule. 

what do you guys think?